docs/config-olap-databases.md

```diff
@@ -4,26 +4,38 @@ title: OLAP Database Configuration
 sidebar_label: OLAP Database
 ---
 
-## Section
+## Introduction
 
-`olap_database:` The OLAP (online analytical processing) database section configures the OLAP database that BuildBuddy uses to enable the Trends page. **Optional**
+To speed up the analysis of historical build event data, BuildBuddy relies on ClickHouse, an OLAP database solution.
+
+Deploying ClickHouse is optional when using BuildBuddy
+BuildBuddy does not require ClickHouse for its core features, such as Build Event Service, Remote Cache, and Remote Execution.
 
-Note: in order to use OLAP database for the Trends page, `app.enable_read_from_olap_db` and
-`app.enable_write_to_olap_db` needs to be set to `true`
+However, some UI features, such as Trends, Drilldown, and Test Grid, do rely on ClickHouse.
```

Comment:
I think the tests page used to work with MySQL and might still work with it? It just doesn't scale as well. Also Drilldown is part of the trends tab so the wording can be simplified

```suggestion
However, some UI features, such as Trends, do require ClickHouse.
```